test/unit/vfocus/mutating.spec.js

```diff
@@ -0,0 +1,129 @@
+var path = require('path');
+var _ = require('lodash');
+var chai = require('chai');
+var expect = chai.expect;
+
+var h = require('virtual-hyperscript');
+var VText = require('vtree/vtext');
+var VFocus = require(path.resolve(process.cwd(), 'src/vfocus'));
+
+
+describe('VFocus - Mutations', function() {
```

Comment:
I agree with your comment. There some tests that does not use the return value but others use it. I'll change them to comply with the **no-return-value** rule.